packages/lesswrong/lib/crud/useQuery.ts

```diff
@@ -1,5 +1,68 @@
+import { createContext, useContext } from "react";
 // eslint-disable-next-line no-restricted-imports
-import { useQuery as useQueryApollo } from "@apollo/client";
+import { useQuery as useQueryApollo, useSuspenseQuery } from "@apollo/client/react";
+import type { SuspenseQueryHookFetchPolicy } from "@apollo/client/react";
+import { debugSuspenseBoundaries, NamedSuspenseBoundary } from "@/components/common/SuspenseWrapper";
+import type { DocumentNode, OperationDefinitionNode, print } from 'graphql';
+import type { TypedDocumentNode } from "@graphql-typed-document-node/core";
 
-export const useQuery: typeof useQueryApollo = useQueryApollo;
+export const EnableSuspenseContext = createContext(false);
 
+type UseQueryOptions = {
+  fetchPolicy: SuspenseQueryHookFetchPolicy,
+  ssr?: boolean,
+  skip?: boolean,
+};
+
+export const useQuery: typeof useQueryApollo = ((query: any, options: UseQueryOptions) => {
```

Comment:
`options` needs to be optional (it is in one of the overloads, and e.g. on `/bestoflesswrong` we experience a crash because we later check `!options.skip` without null-coalescing)

Reply:
Good catch